conduit/data/datasets/vision/waterbirds.py

```diff
@@ -48,7 +48,25 @@ class Waterbirds(CdtVisionDataset[TernarySample, Tensor, Tensor]):
     Split: TypeAlias = WaterbirdsSplit
 
     _BASE_DIR_NAME: ClassVar[str] = "Waterbirds"
-    _METADATA_FILENAME: ClassVar[str] = "metadata.csv"
+    _ORIGINAL_METADATA_FILENAME: ClassVar[str] = "metadata.csv"
+    # Path to the 'fixed' metadata featured in 'MaskTune' and downloaded from the link provided in
+    # the official repo of said paper:
+    # 'https://drive.google.com/file/d/1xPNYQskEXuPhuqT5Hj4hXPeJa9jh7liL/view'.
+    # To quote the paper:
```

Comment:
Not including this information in the docstring because doing so would obligate me to write docstrings for the other parameters and I can't be bothered to do that.